packages/stencilbook/src/stencil-storybook.tsx

```diff
@@ -0,0 +1,226 @@
+import { h as StencilH, FunctionalComponent, VNode } from "@stencil/core";
+import * as Hooks from "@saasquatch/stencil-hooks";
```

Comment:
Should swap to universal hooks

```suggestion
import * as Hooks from "@saasquatch/stencil-hooks";
```